lib/puma/request.rb

```diff
@@ -49,7 +49,7 @@ module Request # :nodoc:
     # @param client [Puma::Client]
     # @param requests [Integer]
     # @return [:close, :keep_alive, :async]
-    def handle_request(client, requests)
+    def handle_request(worker, client, requests)
```

Comment:
1. Based on the naming discussions that have taken place, should this be renamed to processor instead?

    ```suggestion
        def handle_request(processor, client, requests)
    ```
2. Could you add a matching `@param` doc above